DataFormats/TrackerRecHit2D/interface/FastTrackerRecHit.h

```diff
@@ -49,7 +49,7 @@ class FastTrackerRecHit : public BaseTrackerRecHit
     
     /// destructor
     ///
-    ~FastTrackerRecHit() {}
+    virtual ~FastTrackerRecHit() {}
```

Comment:
I agree with Vincenzo. We want to say that the default destructor is to be used here, not that we want to overload the base class destructor. Please change to:
```
~FastTrackerRecHit()=default;
```